content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods.rst

```diff
@@ -0,0 +1,103 @@
+:show-content:
+:hide-page-toc:
+
```

Comment:
No need to include these since there are no sub-docs _yet_ 🙂
```suggestion
```

Reply:
Nice catch! Suggestion implemented ✅